etc/de.rwth.dbis.acis.bazaar.service.BazaarService.properties

```diff
@@ -1,6 +1,3 @@
-dbUserName=reqbaz
-dbPassword=reqbaz
```

Comment:
Removed after an older version of develop branch was merged into changes. Should not cause problems and will be just re-added after merge